micro/walk_test.go

```diff
@@ -6,39 +6,40 @@ import (
 	"github.com/awalterschulze/gominikanren/sexpr/ast"
 )
 
+// u v w x y z
+// 0 1 2 3 4 5
```

Comment:
We could probably get rid of this comment, if we instantiate the variables and pass them into the expressions, because the numbers are much harder to read.